oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/util.test.js

```diff
@@ -0,0 +1,207 @@
+      block: "center",
+    });
+  });
+
+  it("should not throw if no matching element exists", () => {
+    expect(() => scrollToElement("missing.path")).not.toThrow();
+  });
+});
+
+describe("getInputFromDOM", () => {
+  beforeEach(() => {
+    document.body.innerHTML = "";
+  });
+
+  it("should return parsed value from input element with name and value", () => {
+    const input = document.createElement("input");
+    input.setAttribute("name", "testField");
+    input.setAttribute("value", JSON.stringify({ foo: "bar" }));
+    document.body.appendChild(input);
+    const result = getInputFromDOM("testField");
+    expect(result).toEqual({ foo: "bar" });
+  });
+
+  it("should return null if element does not exist", () => {
+    expect(getInputFromDOM("missingField")).toBeNull();
+  });
+
+  it("should return null if element has no value attribute", () => {
+    const input = document.createElement("input");
+    input.setAttribute("name", "noValue");
+    document.body.appendChild(input);
+    expect(getInputFromDOM("noValue")).toBeNull();
+  });
+});
+
+describe("object2array", () => {
+  it("should transform object to array of objects", () => {
+    const obj = { en: "English", fr: "Français" };
+    const result = object2array(obj, "lang", "title");
+    expect(result).toEqual([
+      { lang: "en", title: "English" },
+      { lang: "fr", title: "Français" },
+    ]);
+  });
+});
+
+describe("array2object", () => {
+  it("should transform array of objects to object", () => {
+    const arr = [
+      { lang: "en", title: "English" },
+      { lang: "fr", title: "Français" },
+    ];
+    const result = array2object(arr, "lang", "title");
+    expect(result).toEqual({ en: "English", fr: "Français" });
+  });
+});
+
+describe("requiredMessage", () => {
+  it("should return required message", () => {
+    const label = "Name";
+    const msg = requiredMessage({ label });
+    expect(msg).toContain(label);
+    expect(msg).toContain("is a required field");
+  });
+});
+
+describe("returnGroupError", () => {
+  it("should return group error message", () => {
+    expect(returnGroupError()).toBe("Items must be unique");
+  });
+});
+
+describe("encodeUnicodeBase64 / decodeUnicodeBase64", () => {
+  it("should encode and decode unicode base64", () => {
+    const str = "čřž";
+    const encoded = encodeUnicodeBase64(str);
+    const decoded = decodeUnicodeBase64(encoded);
+    expect(decoded).toBe(str);
+  });
+});
+
+describe("getTitleFromMultilingualObject", () => {
+  it("should return string if input is string", () => {
+    expect(getTitleFromMultilingualObject("Test")).toBe("Test");
+  });
+  it("should return null if input is null", () => {
+    expect(getTitleFromMultilingualObject(null)).toBeNull();
+  });
+  it("should return value for current language", () => {
+    const obj = { en: "English", fr: "Français" };
+    // i18next.language is mocked as 'en' by default
+    expect(getTitleFromMultilingualObject(obj)).toBe(obj.en);
```

Comment:
Try setting i18next.language explicitly for this case (otherwise this breaks when we change global default e.g. to cs), try setting both to en, fr